imap_processing/ultra/l1b/ultra_l1b_extended.py

```diff
@@ -751,31 +750,34 @@ def determine_species(tof: np.ndarray, path_length: np.ndarray, type: str) -> ND
     return species_bin
 
 
-def get_de_az_el(v: NDArray) -> tuple[NDArray, NDArray]:
+def get_theta_phi(
```

Comment:
This is really confusing. It should be `get_phi_theta` because you are returning (phi, theta). Or the return should be reversed.